packages/ui-library/src/components/textarea/index.css.ts

```diff
@@ -43,7 +43,7 @@ export const styleCustom = typeSafeNestedCss`
     justify-content: right;
 
     &.hint, &.error {
-      justify-content: space-between;
+      justify-content: right;
```

Comment:
Thanks for your contribution 😊 This works for the error message, but I noticed that everything gets aligned right when we have both the counter and an error / hint. I was thinking that maybe we could use the :only-child pseudo class to determine if only the counter is there. That way we could use something like this:
`:only-child {
        margin-left: auto;
}
`